src/routes/authRoute.ts

```diff
@@ -18,4 +17,6 @@ router.get('/google/callback', authenticateViaGoogle);
 // Route to login a user
 router.post('/login', login);
 
+router.put('/update-password', updatePassword, isAuthenticated);
```

Comment:
Hey Patrick,
I believe the authenticated middleware should be placed at the beginning of the updatePassword controller.

Reply:
Exactly, please the middleware should be between the route string and the controller function